jobs/aws/eks-anywhere-build-tooling/build-tooling-attribution-files-periodics.yaml

```diff
@@ -15,7 +15,7 @@
 periodics:
   - name: build-tooling-attribution-files-periodic
     # Runs every weekday (M-F) at 2PM PST
-    cron: "0 21 * * 1-5"
+    cron: "0 8 * * 1-5"
```

Comment:
This is the time the eks-d periodics kick off. Should these be moved slightly ahead or behind that time to spread it out?